source/BreachModel.cpp

```diff
@@ -1,17 +1,23 @@
-#include "BreachModel.h"
+﻿#include "BreachModel.h"
+
+/** Default Max Health of a Breach*/
+constexpr unsigned int HEALTH_DEFAULT = 3;
+
 /**
- * Initializes a new breach with the given angle
+ * Initializes a new breach with the given angle and max health
  *
  * This is an initializer.  It, combined with the constructor, produces the static
  * constructor create().  The initializer and normal constructor are private while
  * the static constructor is not.
  *
  * @param  a  The angle at which the breach exists
+ * @param  b  The initial health of the breach
  *
  * @return  true if the obstacle is initialized properly, false otherwise.
  */
-bool BreachModel::init(float a) {
+bool BreachModel::init(float a, int b) {
```

Comment:
Can we make the parameter name a bit more descriptive than just 🅱? I get that `a` is short for `angle`, but what's `b`?